apps/dokploy/templates/zipline/docker-compose.yml

```
@@ -0,0 +1,49 @@
+version: '3'
+services:
+  postgres:
+    image: postgres:15
+    networks:
+      - dokploy-network
+    restart: unless-stopped
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DATABASE=postgres
+    volumes:
+      - pg_data:/var/lib/postgresql/data
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -U postgres']
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  zipline:
+    image: ghcr.io/diced/zipline
+    networks:
+      - dokploy-network
+    ports:
+      - ${ZIPLINE_PORT}
+    labels:
+      - traefik.enable=true
+      - traefik.http.routers.${HASH}.rule=Host(`${ZIPLINE_HOST}`)
+      - traefik.http.services.${HASH}.loadbalancer.server.port=${ZIPLINE_PORT}
+    restart: unless-stopped
+    environment:
+      - CORE_RETURN_HTTPS=${ZIPLINE_RETURN_HTTPS}
+      - CORE_SECRET=${ZIPLINE_SECRET}
+      - CORE_HOST=0.0.0.0
+      - CORE_PORT=${ZIPLINE_PORT}
+      - CORE_DATABASE_URL=postgres://postgres:postgres@postgres/postgres
+      - CORE_LOGGER=${ZIPLINE_LOGGER}
+    volumes:
+      - './uploads:/zipline/uploads'
+      - './public:/zipline/public'
```

Change to this to prevent the folder delete in each deployment
```suggestion
      - '../files/uploads:/zipline/uploads'
      - '../files/public:/zipline/public'
```

---

Can you explain it how adding files directory fixes it?